mult.py

```diff
@@ -0,0 +1,32 @@
+def least_common_multiple(big_num, small_num):
+    count = 1
+    while True:
+        if (big_num * count)%small_num == 0:
+            return big_num * count

```

Comment:
Maybe I don't remember the exercise correctly, but shouldn't you return `small_num * count`?

Reply:
The exercise asked to return the smallest number that can be divided by both input numbers. So I check if the bigger input number * 1 can be divided by the smaller input number, then big_num * 2, etc... The other way around (small_num * 1, small_num * 2...) will produce redundant checks.